internal/performance/export_test.go

```diff
@@ -35,28 +35,43 @@ import (
 	"github.com/google/exposure-notifications-server/pkg/base64util"
 	"github.com/google/exposure-notifications-server/pkg/util"
 	pgx "github.com/jackc/pgx/v4"
+	"github.com/sethvargo/go-envconfig"
 	"github.com/sethvargo/go-retry"
 )
 
+type testConfig struct {
+	Publishes int  `env:"PUBLISHES,required"`
+	Dev       bool `env:"PERFORMANCE_DEV"`
+}
+
 func TestExport(t *testing.T) {
 	const (
 		keysPerPublish = 14
-		// Publishes in 24 hours
-		numPublishes = 100000
 		// Consider the above publishes are evenly distributed in 24 hours, and
 		// period is 10 minutes
-		exportPeriod  = 10 * time.Minute
-		totalBatches  = 24 * 6
-		roughPerBatch = numPublishes/totalBatches + 1
+		exportPeriod = 10 * time.Minute
+		totalBatches = 24 * 6
 	)
 	var (
 		ctx      = context.Background()
 		criteria = publishdb.IterateExposuresCriteria{
 			OnlyLocalProvenance: false,
 		}
+		numPublishes   = 100000
 		want           = keysPerPublish * numPublishes
 		batchStartTime = time.Now().Add(time.Duration(-totalBatches-10) * exportPeriod)
```

Comment:
Move this down to right above line 76 `makoQuickstore, cancel := setup(t)`

Reply:
done